src/gryffin/bayesian_network/model_details.py

```diff
@@ -5,7 +5,7 @@
 #========================================================================
 
 model_details = {
-                'num_epochs':  2 * 10**3, 'learning_rate': 0.05,
+                'num_epochs':  2 * 10**3, 'learning_rate': 0.01,
```

Comment:
Where do the new values for both of these come from?
Actually, where do the old ones come from as well :D

Reply:
The new learning rate comes from my tests. With a smaller learning rate I dont see any parameters duplication. Im not sure why the original learning rate is set this high, might just be copied from Phoenics